ebpf/aya-log-ebpf/src/lib.rs

```diff
@@ -115,6 +115,9 @@ pub fn write_record_header(
         TagLenValue::<RecordField>::new(RecordField::Line, &line.to_ne_bytes()),
         TagLenValue::<RecordField>::new(RecordField::NumArgs, &num_args.to_ne_bytes()),
     ] {
+        if size + attr.value.len() >= buf.len() {
```

Comment:
Same here, the proper check is done in `TagLenValue::write` I think this check isn't needed at all?